sdk/src/feature_set.rs

```diff
@@ -853,6 +853,10 @@ pub mod enable_turbine_extended_fanout_experiments {
     solana_sdk::declare_id!("BZn14Liea52wtBwrXUxTv6vojuTTmfc7XGEDTXrvMD7b");
 }
 
+pub mod deprecate_legacy_vote_ixs {
+    solana_program::declare_id!("depVvnQ2UysGrhwdiwU42tCadZL8GcBb1i2GYhMopQv");
+}
```

Comment:
it seems odd to me to manually backport the creation of this feature for the filtering, but not the actaul code for disallowing the instructions in the processor.
Seems like we should be backporting #587 if we are using this

Reply:
got it - probably fine then, just slightly concerned someone sees the feature is in 2.0, then changes schedule to say it can be activated with 2.0 (which it cannot be).